### ISSUE TITLE
private_key_path option not working

### ISSUE THREAD

**Issue by nickasd** [edamov/pushok#20]
First of all, thanks for creating this library! I never thought it would be so complicated setting up push notifications with the new HTTP/2 protocol.
Although I have a problem with `the private_key_path` option. When providing `__DIR__ . '/key.p8` I get  the following error (I have provided the relative paths only to make it more readable):
```
Notice: Undefined offset: 1 in vendor/spomky-labs/jose/src/KeyConverter/KeyConverter.php on line 190

Fatal error: Uncaught Assert\InvalidArgumentException: Unable to load the key in vendor/beberlei/assert/lib/Assert/Assertion.php:285
Stack trace: #0 vendor/beberlei/assert/lib/Assert/Assertion.php(1679): Assert\Assertion::createException(true, 'Unable to load ...', 38, NULL)
#1 vendor/spomky-labs/jose/src/KeyConverter/KeyConverter.php(158): Assert\Assertion::false(true, 'Unable to load ...')
#2 vendor/spomky-labs/jose/src/KeyConverter/KeyConverter.php(119): Jose\KeyConverter\KeyConverter::loadKeyFromPEM('-\xD19\x8E\x8B\x14\x17/\xAC\xFB!"\xF4-\n...', NULL)
#3 vendor/spomky-labs/jose/src/KeyConverter/KeyConverter.php(103): Jose\KeyConverter\KeyConverter::loadFromKey('-\xD19\x8E\x8B\x14\x17/\xAC\xFB!"\xF4-\xAC...', NULL)
#4 vendor/spomky-labs/jose/src/Factory/JWKFactory.php(347): Jose\KeyConverter\KeyConverter::loadFromKeyFile('key.p8', NULL)
#5 vendor/edamov/pushok/src/AuthProvider/Token. in vendor/beberlei/assert/lib/Assert/Assertion.php on line 285
```
The only way to make it work is adding this code after line 152 in the file `vendor/sponky-labs/jose/src/KeyConverter/KeyConverter.php`:
```
$pem = <<<EOT
<contents of the key.p8 file>
EOT;
```
I'm happy for any suggestion about what might be the problem!

**Comment by edamov** [comment]
Hi @nickasd 
As I see it is an error in library `spomky-labs/jose` or something wrong with your private key. I am not sure that this issue is relative to `pushok` library. So I think you should ask [spomky-labs/jose](https://github.com/Spomky-Labs/jose) first.

**Comment by nickasd** [comment]
Hi @edamov, thanks for your fast response!
I just noticed that the `key.p8` file was somehow corrupted by Ubuntu (on which I am running the php server), so as soon as I replaced it with the one I had saved on my Mac it worked again. gedit on Ubuntu was showing all sorts of strange symbols, so it probably decided to save the file in a different encoding. Now I can see the private key in gedit again.
Thank you again for your help!
